src/App.vue

```diff
@@ -1,8 +1,15 @@
 <template>
   <div>
     <nav class="navbar navbar-expand-lg navbar-light bg-dark">
-      <a id="FiniteGame">FINITE GAME</a>
-      <b-button id="ResetGame" v-on:click="relaodPage">Restart</b-button>
+      <a id="FiniteGame">Finite Quiz</a>
+      <nav class="ms-auto">
+        <b-button class="NavButtons" id="RestartButton" v-on:click="reloadPage">
+          Restart
+        </b-button>
+        <b-button class="NavButtons" id="CloseButton" v-on:click="closeGame">
```

Comment:
You have weird ideas about how classes and IDs should look like…
They should follow [kebab-case](https://gamifyit-docs.readthedocs.io/en/latest/dev-manuals/etc/text-cases.html), and not PascalCase…
Same thing everywhere else

Reply:
done